flake.nix

```diff
@@ -1,4 +1,8 @@
 {
+  # This is needed for ./nix/cli-shell.nix in order to be allowed to have an
+  # impure build.
+  nixConfig.sandbox = "relaxed";
```

Comment:
This is the only line that gets me worried. Is there a possibility it would let us introduce impurity in other derivations in the repo without noticing?

Reply:
Managed to remove it at the cost of having the developers run `--no-sandbox` when doing `nix develop`.